### PR TITLE
Add New Featured Artist: Feint news post

### PR DESCRIPTION
**Pull request by Ephemeralis**
become *feint* with joy, for this is a BIG release 🎵

### CODE REVIEW

File: news/2022-09-28-new-featured-artist-feint.md [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+
+Attempt some classic jumps with [this osu! map](https://osu.ppy.sh/beatmapsets/329290#osu/729986) by [Bearizm](https://osu.ppy.sh/users/3495259)!
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/306/Doxx%20%26%20Feint%20EP%2FDoxx%20%26%20Feint%20-%20The%20Pink%20Fields.mp3" type="audio/mpeg">
+</audio>
+
+### Feint - Vagrant (feat. Veela)
+
+Alternatively, attempt some modern jumps with any of these beatmaps:
```
Comment (Walavouchey) [inline news/2022-09-28-new-featured-artist-feint.md:116]: last time i checked, there were no jumps in taiko

File: news/2022-09-28-new-featured-artist-feint.md [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+
+One more: [this osu! map](https://osu.ppy.sh/beatmapsets/951304#osu/2389865) by [C00L](https://osu.ppy.sh/users/4930630).
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/306/Solace/Feint%20%26%20Laura%20Brehm%20-%20Solace.mp3" type="audio/mpeg">
+</audio>
+
+### Feint - Drifters (feat. Elizaveta)
+
+One more: [this osu! map](https://osu.ppy.sh/beatmapsets/951304#osu/2389865) by [C00L](https://osu.ppy.sh/users/4930630).
```
Comment (Walavouchey) [inline news/2022-09-28-new-featured-artist-feint.md:221]: duplicate line

**Review by Walavouchey — DISMISSED** [review]

File: news/2022-09-28-new-featured-artist-feint.md (hunk 1 above)
Comment (peppy) [reply news/2022-09-28-new-featured-artist-feint.md:116]: Eh, I think it's okay.

File: news/2022-09-28-new-featured-artist-feint.md (hunk 2 above)
Comment (Walavouchey) [reply news/2022-09-28-new-featured-artist-feint.md]: and by that i mean the same map is linked twice

**Review by peppy — APPROVED** [review]

**Pull request merged**